CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added a new Traffic Ops cdn.conf option -- `disable_auto_cert_deletion` -- in order to optionally prevent the automatic deletion of certificates for delivery services that no longer exist whenever a CDN snapshot is taken.
 - [#6034](https://github.com/apache/trafficcontrol/issues/6034) Added new query parameter `cdn` to the `GET /api/x/deliveryserviceserver` Traffic Ops API to filter by CDN name
 - Added a new Traffic Monitor configuration option -- `short_hostname_override` -- to traffic_monitor.cfg to allow overriding the system hostname that Traffic Monitor uses.
+- Added a new Traffic Monitor configuration option -- `stat_polling` (default: false) -- to traffic_monitor.cfg to disable stat polling.
```

Comment:
> i think default is true right? and it is to enable stat polling?

Yes, good catch! I was thinking that the default is usually false for new options, but I specifically wanted this default to be true to avoid double negatives all over the code.

> should we add the distributed polling variable to the CHANGELOG too?

I was going to hold off on adding that one to the changelog until it's actually somewhat functional in a follow-up PR. I just figured it was easier to add both options now even though the `distributed_polling` one will be used in a future PR.